tests/playwright/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# https://playwright.dev/docs/docker
+FROM mcr.microsoft.com/playwright/python:v1.48.0-jammy
```

Comment:
@thekaveman tried running this setup and ran into an error about needing to use the latest image which is now `mcr.microsoft.com/playwright/python:v1.49.0-jammy`.

Is there a `latest` tag we could use? Or maybe it has something to do with the version of `pytest-playwright` (try to pin it)?